packages/contracts-bedrock/src/L2/SuperchainERC20Bridge.sol

```diff
@@ -16,7 +15,7 @@ import { IL2ToL2CrossDomainMessenger } from "src/L2/interfaces/IL2ToL2CrossDomai
 /// @notice The SuperchainERC20Bridge allows for the bridging of ERC20 tokens to make them fungible across the
 ///         Superchain. It builds on top of the L2ToL2CrossDomainMessenger for both replay protection and domain
 ///         binding.
-contract SuperchainERC20Bridge is ISuperchainERC20Bridge, ISemver {
```

Comment:
Why did you remove the `ISemver`?

Reply:
I moved `ISemver` inside `ISuperchainERC20Bridge.sol` because `check-interface` script was failing due to having incompatible ABI between the contract and its interface (`version` was missing at the interface level)

Reply:
I though it was cleaner to put it like that, but also another `version` func can be added

Reply:
perfect then!